util/src/main/java/org/datacommons/util/ReconClient.java

```diff
@@ -0,0 +1,51 @@
+package org.datacommons.util;
+
+import static java.net.http.HttpClient.Version.HTTP_1_1;
+
+import com.google.protobuf.Message;
+import com.google.protobuf.util.JsonFormat;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpRequest.BodyPublishers;
+import java.net.http.HttpResponse.BodyHandlers;
+import org.datacommons.proto.Recon.ResolveCoordinatesRequest;
+import org.datacommons.proto.Recon.ResolveCoordinatesResponse;
+
+/**
+ * Client to the DC resolution APIs.
```

Comment:
Given the general client, maybe have a TODO to refactor https://github.com/datacommonsorg/import/blob/master/util/src/main/java/org/datacommons/util/ExternalIdResolver.java#L296C45-L310 into this?

Reply:
Done.